include/swift/ABI/Metadata.h

```diff
@@ -3706,7 +3721,8 @@ class TargetClassDescriptor final
                                   TargetVTableDescriptorHeader<Runtime>,
                                   TargetMethodDescriptor<Runtime>,
                                   TargetOverrideTableHeader<Runtime>,
-                                  TargetMethodOverrideDescriptor<Runtime>>;
+                                  TargetMethodOverrideDescriptor<Runtime>,
+                                  TargetObjCResilientClassStub<Runtime>>;
```

Comment:
This probably doesn't have to come last since it's not going to be added when back-deploying, but this is good precedent.